packages/mdc-textfield/README.md

```diff
@@ -373,6 +373,11 @@ being used as the text field's "help text". It defaults to `null`. If the text f
 contains an `aria-controls` attribute on instantiation of the component, it will look for an element
 with the corresponding id within the document and automatically assign it to this property.
 
+##### MDCTextfield.helptextValue
+
+String. Proxies to the foundation's `getHelptextValue/setHelptextValue` methods when retrieved/set
```

Comment:
nit: should read `getHelptextValue`/`setHelptextValue`

I realize you were just following the pattern for `MDCTextfield.disabled`, but that should probably change too.

Reply:
Easy fix, thanks!